3rdparty/ittnotify/include/ittnotify.h

```diff
@@ -1711,7 +1711,7 @@ ITT_STUBV(ITTAPI, void, heap_record, (unsigned int record_mask))
  */
 
 /** @cond exclude_from_documentation */
-#pragma pack(push, 8)
+#pragma pack(push, 16)
```

Comment:
This change is not valid because it breaks ABI (this code is just a shim + loader for the actual external implementation)

You should disable ITT for your platform instead (through CMake flags)